### PR TITLE
User Profile attribute translation saves correct key to realm overrides

### LINKED ISSUE #28016: User Profile attribute translation saves wrong key to realm overrides
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

admin/ui

### Describe the bug

When you use the new modal dialog to create translations for a UP attribute display name, it saves a display name of `${profile.attributes.attrName}`.  This part is correct.

However, when the realm override is created or updated, it should not include the `${}` in the key.  It should be just `profile.attributes.attrName`.

### Version

nightly

### Regression

- [ ] The issue is a regression

### Expected behavior

See above

### Actual behavior

See above

### How to Reproduce?

See above

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by agagancarczyk**
Closes https://github.com/keycloak/keycloak/issues/28016

### CODE REVIEW

File: js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx [hunk 1]
```diff
@@ -175,7 +175,7 @@ export const AttributeGeneralSettings = ({
           translationKey={
             editMode
               ? attributeDisplayName
-              : "${profile.attributes." + `${newAttributeName}}`
+              : "profile.attributes." + `${newAttributeName}`
```
Comment (jonkoops) [inline js/apps/admin-ui/src/realm-settings/user-profile/attribute/AttributeGeneralSettings.tsx:178]: Let's use string interpolation for the entire string.
```suggestion
              : `profile.attributes.${newAttributeName}`
```

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
LGTM, let's make sure to backport this.

**Comment by agagancarczyk** [comment]
> LGTM, let's make sure to backport this.

hi @jonkoops this can't be backported as this functionality hasn't been released yet, it will be released in 25

**Comment by jonkoops** [comment]
My bad, I thought this was a fix for something that broke in 24.

**Pull request merged**